src/components/Gnb.tsx

```diff
@@ -31,6 +36,7 @@ export const Gnb = () => {
           {/* <MenuBtn onClick={onClickMenu('/class/' + cid)}>Question List</MenuBtn> */}
           <MenuBtn onClick={onClickSwitchClass}>Switch Class</MenuBtn>
           <MenuBtn onClick={onClickMyPage}>My Page</MenuBtn>
+          {isAdmin && cid && <MenuBtn onClick={onClickAdmin}>Admin</MenuBtn>}
```

Comment:
```suggestion
          {isAdmin && <MenuBtn onClick={onClickAdmin}>Admin</MenuBtn>}
```
could you change `onClickAdmin` to check cid value instead?

Reply:
> could you change `onClickAdmin` to check cid value instead?

Alright:)